recipes/legit/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set packagename = 'legit' %}
+{% set version = "0.2.0" %}
+{% set md5 = "5df3cd402b0de8f0251a87904b5eb88c" %}
+
+package:
+  name: {{ packagename }}
+  version: {{ version }}
+
+source:
+  fn: {{ packagename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ packagename[0] }}/{{ packagename }}/{{ packagename }}-{{ version }}.zip
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - legit = legit.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+  run:
+    - python
+    - clint >=0.4.1
+    - gitpython >=0.3.4
+    - six >=1.9.0
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - {{ packagename }}
+
+  commands:
+    - legit --help
```

Comment:
So I agree with @msarahan we should not be assuming that `SRC_DIR` exists in tests. It might not be the case. Even if it does exist it, it can confound problems as it did here. His advice to clone some other repo for testing seems sound. Please also add `git` as a test dependency.